src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,44 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_SIZE = 10;
+    private int size = 0;
+    private final K[] keys;
+    private final V[] values;
```

Comment:
final fields should be before non finals
also size will be 0 by default
```suggestion
    private final K[] keys;
    private final V[] values;
    private int size;
```